### PR TITLE
Propagate logging level to scanner and add remove_service method

### LINKED ISSUE #46: `Castlistner` should implement `remove_service`
**Describe the bug**

```text
2021-03-21 10:56:13 ERROR (zeroconf-ServiceBrowser__googlecast._tcp.local._264) [root] Uncaught thread exception
Traceback (most recent call last):
  File "/usr/local/lib/python3.8/threading.py", line 932, in _bootstrap_inner
    self.run()
  File "/usr/local/lib/python3.8/site-packages/zeroconf/__init__.py", line 1760, in run
    self._service_state_changed.fire(
  File "/usr/local/lib/python3.8/site-packages/zeroconf/__init__.py", line 1513, in fire
    h(**kwargs)
  File "/usr/local/lib/python3.8/site-packages/zeroconf/__init__.py", line 1618, in on_change
    listener.remove_service(*args)
AttributeError: 'CastListener' object has no attribute 'remove_service'
```

See this link [Homeassistant/core#48117](https://github.com/home-assistant/core/issues/48117)

**To Reproduce**

Appears first time i restart, but only once.

**Expected behavior**

No error message.

**Screenshots**

**Additional context**

### PR DESCRIPTION
**Pull request by leikoilja**
Closes #46

### CODE REVIEW

File: glocaltokens/scanner.py [hunk 1]
```diff
@@ -28,22 +28,26 @@ def count(self):
...
+        LOGGER.debug("update_service %s, %s", typ, name)
         self._add_update_service(zconf, typ, name, self.update_callback)
 
-    def add_service(self, zconf, typ, name):
-        """ Add a service to the collection. """
-        _LOGGER.debug("add_service %s, %s", typ, name)
-        self._add_update_service(zconf, typ, name, self.add_callback)
+    def remove_service(self, _zconf, typ, name):
+        """Called when a cast has beeen lost (MDNS info expired or host down)."""
+        LOGGER.debug("remove_service %s, %s", typ, name)
```
Comment (leikoilja) [inline glocaltokens/scanner.py:43]: Don't think we actually need to do anything here at this point.
If I understand it right this one is more like a `callback` when `mdns` info expires or host is down.

**Review by KapJI — APPROVED** [review]
In the title: `logging`?

👍 for renaming `_LOGGER`.

File: glocaltokens/scanner.py [hunk 2]
```diff
@@ -28,22 +28,26 @@ def count(self):
...
-        _LOGGER.debug("update_service %s, %s", typ, name)
+        LOGGER.debug("update_service %s, %s", typ, name)
         self._add_update_service(zconf, typ, name, self.update_callback)
 
-    def add_service(self, zconf, typ, name):
-        """ Add a service to the collection. """
-        _LOGGER.debug("add_service %s, %s", typ, name)
-        self._add_update_service(zconf, typ, name, self.add_callback)
+    def remove_service(self, _zconf, typ, name):
+        """Called when a cast has beeen lost (MDNS info expired or host down)."""
```
Comment (KapJI) [inline glocaltokens/scanner.py:42]:
```suggestion
        """Called when a cast has beeen lost (mDNS info expired or host down)."""
```

**Comment by leikoilja** [comment]
oh, you're right, i meant `logging` level 😅

**Comment by leikoilja** [comment]
Thanks, @KapJI

**Comment by leikoilja** [comment]
@DurgNomis-drol, the newest version of `ha-google-home` (will do a release later tonight) will be using the newest version of `glocaltokens` that will have the fix for `remove_service`. Please keep an eye on the logs that it fixes the issue you were having :)

**Comment by DurgNomis-drol** [comment]
I will test it when you have released a new version 😊 I am out of the house to night, but i will look into to this tomorrow

**Pull request merged**